frontend/src/App.jsx

```diff
@@ -20,6 +20,7 @@ import { UserProvider } from './helpers/UserContext';
 import MarketDetails from './pages/marketDetails/MarketDetails';
 import User from './pages/user/User';
 import Footer from './components/footer/Footer';
+import '../index.css';
```

Comment:
OK so it's implied that we will import index.css in every page where we are using tailwind now or does importing it into App.jsx cover it for every page?